client/postit/src/App.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable react-hooks/rules-of-hooks */
+import axios from "axios";
+import SignUp from "./components/SignUp";
+import SignIn from "./components/SignIn";
+import API_ROUTES from "./utils/constants";
+import { useState } from "react";
+
+function App() {
+  const [displaySignIn, setDisplaySignIn] = useState(false);
+
+  const handleRegistration = async (username, password, email) => {
+    try {
+      const response = await axios.post(API_ROUTES.SIGN_UP, {
+        username,
+        password,
+        email,
+      });
+
+      if (response.status === 201){
```

Comment:
Could we have this method also return something when its successful something like {message: success, status: response.status}, then for errored response something like this {message: 'error', status: response.status}. That way, it's more uniform in the places it's called . And more readable to someone seeing the code for the first time